### ISSUE TITLE
x/build/cmd/releasebot: consider a third -mode=wrapup mode for wrapping up a release

### ISSUE THREAD

**Issue by dmitshur** [golang/go#36086]
`releasebot` has two modes: prepare, and release. Prepare does some preparatory work, while release performs the actual work of releasing.

We can consider adding a third mode, to be run at the end of a release and ensure humans did not forget some post-release steps.

### Background

There are some tasks done by release managers after `-mode=release`, including sending a CL to create a `golang.org/dl/goX.Y.Z` command (e.g., [CL 204559](https://golang.org/cl/204559)), making a tweet (e.g., https://twitter.com/golang/status/1190073956876849152), redeploying the golang.org website, etc. After #36075 is finished, it can also check that x/website has the latest minor release documented on the [Release History](https://golang.org/doc/devel/release.html) page; something that used to be done as part of prepare mode.

While some of those can and should be automated over time, at this time there are non-zero such tasks. We can consider creating a third mode to be run at the very end of a release, and its purpose would be to perform checks that all the post-release steps have been completed.

In the past, we've relied on release managers not forgetting/skipping steps in the release process, and people reporting issues when we do.

/cc @cagedmantis @toothrot @bradfitz

**Comment by dmitshur** [comment]
[CL 221097](https://golang.org/cl/221097) deletes the following check from `Work.checkDocs` method, because it cannot be done as part of the prepare or release modes as it was done before (it's too soon):

```Go
// TODO: Delete this block after Go 1.14 is released (and Go 1.12 is no longer supported),
//       or consider moving it into a third -mode=wrapup. See golang.org/issue/36086.
if w.activeReleaseHistory() {
	// Check that the release is listed on the release history page.
	data, err := ioutil.ReadFile(filepath.Join(w.Dir, "gitwork", "doc/devel/release.html"))
	if err != nil {
		w.log.Panic(err)
	}
	if !strings.Contains(string(data), w.Version+" (released ") {
		w.logError("doc/devel/release.html does not document %s", w.Version)
	}
}
```

If a third wrapup mode is added, it can be re-added as part of that mode.

**Comment by dmitshur** [comment]
This is a key part of what relui does for us, and by now it has superseded releasebot. Closing.
